### ISSUE TITLE
Trained the Balloon model, but couldn't replicate the .h5 file in the release page

### ISSUE THREAD

**Issue by sammiya** [matterport/Mask_RCNN#1881]
I runned `samples/balloon/balloon.py` as below.
```
git clone https://github.com/matterport/Mask_RCNN.git
cd Mask_RCNN/samples/balloon/
python3 balloon.py train --dataset=balloon --weights=coco
```
After 20 hours, `mask_rcnn_balloon_0030.h5` was generated in `Mask_RCNN/logs/balloon20191117T0132`. However, color splash effect with the .h5 file doesn't work as well as that with `mask_rcnn_balloon.h5` downloaded from https://github.com/matterport/Mask_RCNN/releases .

Here's an example (used https://gyazo.com/2ffec6f8292323302b9bd2a09eb24338 as input):

- result using `mask_rcnn_balloon.h5` (downloaded from https://github.com/matterport/Mask_RCNN/releases): https://gyazo.com/1724917302a49d82e0abb789ff6b955d

- result using `mask_rcnn_balloon_0030.h5` (which I generated): https://gyazo.com/95a9f55d1e87b8bb48532b6ceca3ffa0

Any ideas? Do I need to increase epoch number (30) or step number (100) in order to replicate the `mask_rcnn_balloon.h5` file?

My machine environment:

Ubuntu 16.04.6 LTS
CUDA 9.1
Python 3.6.8
tensorflow 1.14.0
keras 2.2.5

**Comment by marcfielding1** [comment]
Err don't you have to convert the mode, one sec lemmy check..

**Comment by marcfielding1** [comment]
Do you have your training graph what were your loss/validation values when you stopped it at 30 epochs? 

Not that this matter but what were you training that on a CPU?

**Comment by marcfielding1** [comment]
I've just reread this, the checkpoints(the h5 files) end up in /logs ! Sorry I scan read the first time,

**Comment by cc-bundo** [comment]
Thank you for your reply, and I'm terribly sorry for the late response.
After checking the output of the loss/validation graphs, the graphs did not seem to converge, so I increased the epoch number, and finally got as good splash effect as sample. Sorry for bothering you.

**Comment by sammiya** [comment]
Thank you for your reply, and I'm terribly sorry for the late response.
After reading your comment, I checked the output of the loss/validation graphs and the graphs did not seem to converge, so I increased the epoch number, and finally got as good splash effect as sample. Thank you so much for that.
